meetings/2020-08-13.md

```diff
@@ -0,0 +1,80 @@
+﻿# Node.js Technical Steering Committee (TSC) Meeting 2020-08-13
+
+## Links
+
+* **Recording**:  https://youtu.be/XV6AI4AdnM0
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/907
+
+## Present
+
+* Ruben Bridgewater @BridgeAR (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Mary Marchini @mmarchini (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Robert Nagy (Invited guest)
+
+## Agenda
+
+
+### Announcements
+
+* Mary: Change to collaborator guide, if you object make sure to use the requests changes to indicate you don’t want it to land unless your comments/objections are addressed
+* Mary: Survey on unhandled rejections is up, please participate. ~ 500 participants so far, but would like about 2000 before closing, if we get answers at same rate we should meet that.
+* Mary: Commit queue pull request will land later today, will let us land using a label. Once it lands the Triage team will be able to land PRs as well, but only if it meets all of the requirements so it should be safe. If you object please say so on meeting issue or the pull request.
+
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+
+### nodejs/node
+
+
+* Land #34035 on v14 (stream: simpler Readable async iterator) [#34680](https://github.com/nodejs/node/issues/34680)
+  * implementation labelled as SemVer major, done defensively. Would like to port back to 14.x
+  * async iterator, changed to be based on async generator
+  * Ways that it is breaking
+    * Error handling
+    * prototype
+  * Believe it is very unlikely that these changes would affect existing code.  Strange to call next
+     after it has already failed
+  * Myles, have tests been run with CITGM? To be comfortable:
+    * open backport PR
+    * run CITGM
+    * reach out to those who have deep experience in addition to Ronag (Matteo, etc.)
+    * reach out to some key ecosystem
+      * Myles, possibly Feros, will give some other suggestions.
+    * Let’s try to get it in before 14 goes to LTS.

```

Comment:
This was the consensus/decision? (I support that. I just want to know if the matter is settled.)

Reply:
@Trott there were only 5 in the meeting, but those in the meeting did not raise any objections so we can assume consensus from that group.